Dockerfile

```diff
@@ -79,11 +82,12 @@ COPY --from=builder --chown=nextjs:nodejs /app/prisma ./prisma
 
 COPY --chown=nextjs:nodejs cron.js ./cron.js
 COPY --chown=nextjs:nodejs entrypoint.sh ./entrypoint.sh
+RUN chmod +x ./entrypoint.sh
 
 USER nextjs
 
 # Default port to 3000
 ENV PORT 3000
 
 # CMD ["node", "server.js"]
-CMD ["/bin/sh", "entrypoint.sh"]
+CMD ["dumb-init", "--", "./entrypoint.sh"]
```

Comment:
Was macht `dumb-init`?

Reply:
I think best practice would be to have the migration on the docker-compose level as it is more obvious to an engineer that this migration step is happening with each docker container start. However, i understand it is easier here as we have access to prisma here. Hence i think it is ok.

Reply:
See description: signal forwarding using dumb-init
- eg ctrl-c or shutdown requests. Nodejs doesn't handle them well when run as PID 1

Reply:
Goal: migrations work as part of the startup of the container wie package. Needs to also work in the non-docker-compose workflow when deploying the container